tests/test_playwright.py

```diff
@@ -1,4 +1,8 @@
-"""Build minimal test sites with sphinx_build_factory and test them with Playwright."""
+"""
+Build minimal test sites with sphinx_build_factory and test them with Playwright.
+When adding new tests to this file, remember to also add the corresponding test site
+to `tests/sites/` or use an existing one.
```

Comment:
This is fine for now, but the advice "or use an existing one" might change depending on what others think of my (admittedly opinionated) suggestions to whittle down to a bare-bones one-purpose-per-test-site approach.

Reply:
TBF I like your proposal of simplifying the tests by reducing the number of test sites. 
I added this comment for now as we struggled at first to identify why the site was not being built or on what sites we were running tests